guides/handGrenades.md

```diff
@@ -0,0 +1,107 @@
+# Hand Grenade Guide
+
+## Role Description
+
+All operators should be expected to familiarize themselves with the use of hand grenades. This guide will cover mainly fragmentation grenades, but will briefly cover the other 2 types as well.
+
+## Specialized Equipment
+
+Completing the certification for hand grenades will unlock the usage of fragmentation grenades as well as flashbang grenades.
+
+## Usage
+
+### Callouts
+
+Similar to throwing a grenade, every time you toss a grenade, a call should be made over proximity voice. The call should be in the format:
+
+> [Frag|Flash|Smoke] out. [Direction]
+
+Examples:
+
+> Frag Out! North!
+> Flash Out! South-East!  
+> Smoke Out! West! 
+
+The call out should not slow down your ability to respond to threats. At a minimum you must always call out "[Grenade Type] Out!"
+
+If the employment of the grenade fails, the thrower should immediately alert the other operators in his vicinity.
+
+Example:
+
+> Bad 'nade!
+> Bad frag! 
+> Bad flash! Get down! 
+
+### Risks of Use
+
+When firing throwing fragmentation grenades, you need to know the impact radius and fragmentation 
+radius to maximize injury on the enemy and reduce friendly injuries.
+
+The approximate lethality of an M67 fragmentation grenade is:
+
+- 5m - Deadly
+- 15m - Casualty radius
+- 230m - Fragmentation radius (shrapnel can disperse up to this far.)
+- 35m - Range an average soldier can throw
+
+Regardless of range there is always a chance of being hit by a piece of shrapnel. 
+Due to this risk, proper employment should always consist of throwing the grenade and immediately seeking cover.
+
+#### Vehicles
```

Comment:
This should include a note not to attempt to use a grenade on any military vehicles, even the lightest armor will render them useless